web/pom.xml

```diff
@@ -30,6 +30,12 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
+            <groupId>no.nav.openapi.spec.utils</groupId>
+            <artifactId>openapi-spec-utils</artifactId>
+            <version>${openapi-spec-utils.version}</version>
```

Comment:
Vi er på EE11 og dermed REST 4. 
openapi-spec-utils har satt limit til JaxRs 3.9 (jakarta.ws.rs) - den bør opp til V4 som vi bruker

Reply:
Ditto gjelder for jakarta.annotation. Der er vi på V3

Reply:
Hva skal til for at det bumpes i openapi-spec-utils @josstn ?

Reply:
Oppdatert 👍